Specs/DateTime/German/DateTimePeriodParser.json

```diff
@@ -21,7 +21,7 @@
           }
         },
         "Start": 8,
-        "Length": 26
+        "Length": 25
```

Comment:
What will happen if we do have a whitespace at the end of the query?

Reply:
It will return the same result without the whitespace. I believe this is the expected behavior so I changed this in the spec.